app/src/main/java/space/taran/arkretouch/presentation/edit/MoreOptionsPopup.kt

```diff
@@ -0,0 +1,72 @@
+package space.taran.arkretouch.presentation.edit
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Popup
+import space.taran.arkretouch.R
+
+@Composable
+fun MoreOptionsPopup(
+    onShareClick: () -> Unit,
+    onDismissClick: () -> Unit
+) {
+    Popup(
+        alignment = Alignment.TopEnd,
+        offset = IntOffset(-36, 36),
+        onDismissRequest = onDismissClick,
+    ) {
```

Comment:
Let's also add
`properties = PopupProperties(focusable = true),`
to avoid an unnecessary dot on canvas when trying to close popup